pkg/pipelines/tasks/deploy_from_source_task.go

```diff
@@ -0,0 +1,72 @@
+package tasks
+
+import (
+	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha2"
+)
+
+// generateDeployFromSourceTask will return a github-status-task
+func generateDeployFromSourceTask() pipelinev1.Task {
+	task := pipelinev1.Task{
+		TypeMeta:   createTaskTypeMeta(),
+		ObjectMeta: createTaskObjectMeta("deploy-from-source-task"),
+		Spec: pipelinev1.TaskSpec{
+			Inputs: createInputsForDeployFromSourceTask(),
+			TaskSpec: v1alpha2.TaskSpec{
```

Comment:
you're using `v1alpha1` and `v1alpha2` in the same code.

Reply:
Resolving this since the latest version of v1alpha1 uses v1alpha2 for `taskSpec`.